examples/zephyr/README.md

```diff
@@ -1,34 +1,118 @@
-# Golioth Zephyr Examples
+# Golioth Firmware SDK: Zephyr Support (beta)
 
-This directory contains a set of examples intended to build and run with the latest release of
-Zephyr.
+Zephyr support in the Golioth Firmware SDK is in active development and
+currently designated as "beta". We will update this page when this port
+is ready for general availability. For production devices, please use
+the [Golioth Zephyr SDK](https://github.com/golioth/golioth-zephyr-sdk).
 
-### Install Zephyr SDK
+The SDK contains a set of examples designed to run using Zephyr
+RTOS.
 
-Follow [Zephyr Getting Started Guide](https://docs.zephyrproject.org/latest/develop/getting_started/).
+API documentation: <https://firmware-sdk-docs.golioth.io/>
 
-### Setup west project
+## Install Golioth Firmware SDK with Zephyr RTOS Support
```

Comment:
I think it might be clearer to re-order these headings/sections, so that using this repo as the manifest repo (for either Zephyr or NCS) are grouped together, and adding the SDK to an existing west manifest is separated out (because it works for both NCS and Zephyr, it doesn't make sense to only have it under Zephyr). Here's what I'm thinking

## Using the Golioth Firmware SDK as a West manifest repository

### Using with mainline Zephyr

### Using with Nordic's nRF Connect SDK

## Adding the Golioth Firmware SDK to an existing Zephy-based West project